tools/check-content.sh

```diff
@@ -1,17 +1,39 @@
 #!/bin/sh -e
 
-# Check Go source files for disallowed content.
+# Check Go source files for disallowed or stale content.
 
-echo 'Disallowed import of net/rpc:'
-if find . -name '*.go' | xargs grep '"net/rpc"'; then
+# Verify the generated code is up to date.
```

Comment:
This new check doesn't really belong in this script. It is very different from the existing checks, it depends on Git and requires a clean Git repo.